runtime/darwinia/src/pallets/system.rs

```diff
@@ -54,7 +53,6 @@ frame_support::parameter_types! {
 		})
 		.avg_block_initialization(AVERAGE_ON_INITIALIZE_RATIO)
 		.build_or_panic();
-	pub const SS58Prefix: u8 = 42;
```

Comment:
Can we put it in primitives? It is also needed in chain_spec

Reply:
Move this to #66.

TODO